build/buildkite-build.sh

```diff
@@ -10,7 +10,6 @@ cd `dirname $0`/..
 echo "--- :scala: Compiling main codebase"
 /usr/local/bin/sbt                  \
   -jvm-opts build/buildkite-jvmopts \
-  -Docs3.skipDependencyUpdates      \
```

Comment:
I think we can completely delete this file

Reply:
Yeah there's probably a lot of cleanup that can be done now but I don't want to hold this up anymore because I have to start over if any of the deleted code changes in `develop`.